quantinuum_schemas/models/quantinuum_systems_noise.py

```diff
@@ -44,3 +45,16 @@ class UserErrorParams(BaseModel):
     emission_scale: Optional[float] = None
     crosstalk_scale: Optional[float] = None
     leakage_scale: Optional[float] = None
+
+
+class SeleneErrorMode(str, Enum):
+    """Mode for error modeling in the Selene emulator."""
+
+    NONE = "NONE"
+    SIMPLE = "SIMPLE"
+    FULL = "FULL"
+    CUSTOM = "CUSTOM"
+
+
+class SeleneCustomNoiseModel(BaseModel):
+    """A custom noise model for the Selene emulator."""
```

Comment:
Assuming here that the only error mode with user configuration is the custom one.